### PR TITLE
Update Text file18 05 .txt

### PR DESCRIPTION
**Pull request by cafeaua**
Edited for a new branch.

### CODE REVIEW

File: Text file18 05 .txt [hunk 1]
```diff
@@ -1 +1,2 @@
 This is the first text file. 
+I edit this so I can add colaborators.
```
Comment (cafeaua) [inline Text file18 05 .txt]: Comment on the line 18 08 Cafeaua

**Comment by SiposZoltan** [comment]
What is here? 19 48

**Review by SiposZoltan — COMMENTED** [review]
It is uncool satisfyingly. 19 48

**Comment by SiposZoltan** [comment]
Close pul req 19 49

**Comment by SiposZoltan** [comment]
Closing pull request 19 50

File: Text file18 05 .txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 This is the first text file. 
 I edit this so I can add more (18 09) colaborators.
+AAAA...19 41
```
Comment (cafeaua) [inline Text file18 05 .txt]: In line comment 19 51 heheh caf

**Review by cafeaua — COMMENTED** [review]
Comment comment 19 51

**Pull request closed without merging**